kuksa-client/kuksa_client/grpc/aio.py

```diff
@@ -251,19 +255,23 @@ async def set(self, *, updates: Collection[EntryUpdate], **rpc_kwargs) -> None:
                 grpc.*MultiCallable kwargs e.g. timeout, metadata, credentials.
         """
         paths_with_required_type = self._get_paths_with_required_type(updates)
+        
         paths_without_type = [
             path for path, data_type in paths_with_required_type.items() if data_type is DataType.UNSPECIFIED
         ]
         paths_with_required_type.update(await self.get_value_types(paths_without_type, **rpc_kwargs))
         req = self._prepare_set_request(updates, paths_with_required_type)
+        metadata = []
+        metadata.append(("authorization", self.token))
+        rpc_kwargs.__setitem__("metadata", metadata)
         try:
             resp = await self.client_stub.Set(req, **rpc_kwargs)
         except AioRpcError as exc:
             raise VSSClientError.from_grpc_error(exc) from exc
         self._process_set_response(resp)
 
-    async def authorize(self, *, token: str) -> str:
-        raise NotImplementedError('"authorize" is not yet implemented')
+    async def authorize(self, *, token: Dict) -> None:
+        self.token = "Bearer " + token
```

Comment:
Calling it `self.token` is confusing as it doesn't actually store the token.
It's fine to construct the authorization header here (instead of doing it repeatedly later), but in that case it should probably be called `self.authorization_header` or something like that instead.